wikipedia-dark.user.css

```diff
@@ -3721,6 +3762,7 @@
   table.wikitable,
   button.formBtn {
     border: 1px solid var(--gray-5) !important;
+    background-color: var(--gray-2) !important;
   }
```

Comment:
This section is handling borders, so adding rules affecting backgrounds is not appropriate here (and will cause collateral damage). A more specific selector should be chosen for this rule as well.